src/Microsoft.NetCore.Analyzers/UnitTests/Data/ReviewSQLQueriesForSecurityVulnerabilitiesTests.cs

```diff
@@ -1,23 +1,30 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System;
-using Microsoft.CodeAnalysis.Diagnostics;
+using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.Testing;
 using Test.Utilities;
+using Test.Utilities.MinimalImplementations;
 using Xunit;
+using VerifyCS = Test.Utilities.CSharpSecurityCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Data.ReviewSqlQueriesForSecurityVulnerabilities,
+    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
```

Comment:
We have been using a lot this syntax with the `EmptyCodeFixProvider`, would it be worth having a sub-class to `CSharpSecurityCodeFixVerifier` with only the analyzer as generic type argument? On one side that would make it easier for new rule but on the other side that means that there are 2 ways to declare this static using. WDYT?

Reply:
I think it's fine like this. It helps with two things:

1. Encourage creating a code fix with an analyzer as the recommended practice
2. Discourage separating analyzer tests from code fix tests in separate files